src/main/java/com/connexta/ingest/controllers/IngestController.java

```diff
@@ -33,19 +33,16 @@ public IngestController(IngestService ingestService) {
 
   @Override
   public ResponseEntity<Void> ingest(
-      String acceptVersion, MultipartFile multipartFile, String correlationId) {
-    String mediaType = multipartFile.getContentType();
-    String fileName = multipartFile.getOriginalFilename();
-    Long fileSize = multipartFile.getSize();
-    log.info(
-        "Ingest request received acceptVersion={}, fileName={}, fileSize={}, mediaType={}",
-        acceptVersion,
-        fileName,
-        fileSize,
-        mediaType);
+      String acceptVersion, MultipartFile file, String correlationId, MultipartFile metacard) {
+    String mediaType = file.getContentType();
+    String fileName = file.getOriginalFilename();
+    Long fileSize = file.getSize();
+    String metacardName = metacard.getOriginalFilename();
+    log.info("Ingest request received fileName={}", fileName);
+    log.info("Ignoring attached metacard: {}", metacardName);
```

Comment:
✏️ I don't think the name of the metacard file is useful. `log.info("Ignoring attached metacard)` is good enough, I think.

Reply:
I agree, we won't have this message up for more than a couple weeks anyway.